infrastructure/tests/test_data_import.py

```diff
@@ -143,7 +143,7 @@ def test_file_upload(self):
     def test_file_upload_fail(self):
         """Scope of Test: Testing if the file upload fail in Django Admin to processing file and add to Django_Q"""
         self.client.login(username=self.username, password=self.password)
-        fy = FinancialYear.objects.create(budget_year="2019/2020", active=1)
+        fy, _ = FinancialYear.objects.get_or_create(budget_year="2019/2020")
```

Comment:
Agreed. I actually think it's important to fix this because an ambiguousness like this could make the test pass even though the code is broken. Changing to use get() because we expect a year to exist